criu/uffd.c

```diff
@@ -284,6 +284,7 @@ int uffd_open(int flags, unsigned long *features)
 	}
 
 	if (uffdio_api.api != UFFD_API) {
+		/* coverity[format_error] */
```

Comment:
I think we need to support each annotation with a comment that explains why we think that it is a false positive case.